splice_spark/pom.xml

```diff
@@ -224,8 +224,32 @@
                 <version>1.5.0.Final</version>
             </extension>
         </extensions>
-        <plugins>
-            <plugin>
+	<plugins>
+		<plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-site-plugin</artifactId>
+			<version>3.0</version>
+			<configuration>
+				<reportPlugins>
+					<plugin>
+						<artifactId>maven-project-info-reports-plugin</artifactId>
+						<version>2.2</version>
+					</plugin>
+					<plugin>
+						<groupId>net.alchim31.maven</groupId>
+						<artifactId>scala-maven-plugin</artifactId>
+						<version>3.4.2</version>
+						<configuration>
+							<jvmArgs>
```

Comment:
I'm curious about the number of spaces for indentation used? 8? Why not 4? Even for such a simple change we're almost at 144-char limit (!)